test/functional/apps/visualize/_visualize_listing.ts

```diff
@@ -45,7 +45,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     describe('search', function () {
       before(async function () {
         // create one new viz
-        await PageObjects.visualize.gotoVisualizationLandingPage();
         await PageObjects.visualize.navigateToNewVisualization();
```

Comment:
`gotoVisualizationLandingPage` is actually the first step of `navigateToNewVisualization`